src/common/ipfixlolib/ipfixlolib.h

```diff
@@ -125,9 +125,16 @@ See ipfixlolib.h for details on how to use this library.
 extern "C" {
 #endif
 
+/* Supported export protocols */
+typedef enum export_protocol_version {
```

Comment:
And actually if you look at the place ipfix_init_exporter() in the diffs is called it horrible mixes the enum and the defines. Given the enums have the correct values, it would be sufficient to just use them instead. Alternative to just use the defines, the functions would have be changed to just take uint. I'm not sure I like that as much, as it really does denoting the type of export protocol behaviour and isn't really just a number to insert in the header.
ABI would be fine as long enum values are only ever added, and these existing standardised values are never going to go away, so I don't see a significant problem. For that matter adding another export version (IPFIX 11 anyone? :-) ) also seem extremely remote.